src/osgPlugins/rot/ReaderWriterROT.cpp

```diff
@@ -80,7 +80,7 @@ static bool getFilenameAndParams(const std::string& input, std::string& filename
  * This pseudo-loader makes it simple to change the orientation of a saved
  * model by specifying a correcting rotation as part of the filename.
  *
- * Usage: <modelfile.ext>.<rx>,<ry>,<rz>.globe
```

Comment:
Why this change and why is part of this PR?

Reply:
This is correcting what looks like a copy and paste error in the comment describing the usage of the ROT plugin.